modules/infrastructure-components.adoc

```diff
@@ -1,21 +1,21 @@
 // Module included in the following assemblies:
 //
 // * machine_management/creating-infrastructure-machinesets.adoc
-// * post_installation_configuration/cluster-tasks.adoc
 // * nodes-nodes-creating-infrastructure-nodes.adoc
 
 [id="infrastructure-components_{context}"]
 = {product-title} infrastructure components
 
-The following infrastructure workloads do not incur {product-title} worker subscriptions:
+Each self-managed Red Hat OpenShift subscription includes entitlements for {product-title} and other OpenShift-related components. These entitlements are included for running {product-title} control plane and infrastructure workloads and do not need to be accounted for during sizing.
```

Comment:
Nit - add a {nbsp} between Red and Hat per [RHSSG](https://redhat-documentation.github.io/supplementary-style-guide/#non-breaking-spaces):
```suggestion
Each self-managed Red{nbsp}Hat OpenShift subscription includes entitlements for {product-title} and other OpenShift-related components. These entitlements are included for running {product-title} control plane and infrastructure workloads and do not need to be accounted for during sizing.
```